modules/ingestor/src/service/mod.rs

```diff
@@ -180,8 +180,8 @@ impl IngestorService {
         match fmt {
             Format::SPDX | Format::CycloneDX => {
                 let analysis_service = AnalysisService::new(self.graph.db.clone());
-                match analysis_service
-                    .load_graphs(vec![result.id.value()], ())
+                match analysis_service // TODO: today we chop off 'urn:uuid:' prefix using .split_off on result.id
+                    .load_graphs(vec![result.id.to_string().split_off(9)], ())
```

Comment:
To be more robust, maybe something guarded with starts_with("urn:uuid:") and then munging, probably also with "urn:uuid:".len() to make it extremely clear which 9 characters we're concerned about?